packages/app/app/components/PlayQueue/QueueMenu/QueueMenuMore/index.js

```diff
@@ -33,7 +34,7 @@ const QueueMenuMore = ({
     <Dropdown item icon='ellipsis vertical' className={styles.queue_menu_more} disabled={disabled}>
       <Dropdown.Menu>
         <Dropdown.Header>{t('header')}</Dropdown.Header>
-        <Dropdown.Item onClick={clearQueue}>
+        <Dropdown.Item onClick={() => {clearQueue();resetPlayer(sendPaused);}}>
```

Comment:
You can use `withHandlers` in `compose` for this, e.g.

```javascript
handleClearClick: ({ clearQueue, resetPlayer, sendPaused }) => () => {
  clearQueue();
  resetPlayer(sendPaused);
}
```

Just remember to move your `withProps` to the top inside the compose function so that the handler has access to `sendPaused`.